maintainers/maintainer-list.nix

```diff
@@ -6649,6 +6649,12 @@
     email = "nfjinjing@gmail.com";
     name = "Jinjing Wang";
   };
+  ngiger = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add ngiger``.